plugins/CuraEngineBackend/CuraEngineBackend.py

```diff
@@ -54,7 +54,7 @@ def __init__(self):
         if Platform.isWindows():
             default_engine_location += ".exe"
         if Platform.isLinux() and not default_engine_location:
-            if not os.getenv('PATH'):
+            if not os.getenv("PATH"):
                 raise OSError("There is something wrong with your Linux installation.")
             for pathdir in os.getenv('PATH').split(os.pathsep):
```

Comment:
I hate that this ```'``` key is so easy to type on QWERTY keyboards. Always using ```'``` because it is comfortable ^^